charts/drax/charts/network-state-monitor/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: network-state-monitor
 description: The Accelleran dRAX Network State Monitor
 type: application
-version: 0.1.0
+version: 0.2.0
 # renovate: image=accelleran/network-state-monitor
-appVersion: 1.1.0
+appVersion: 1.2.0
 dependencies:
   - name: common
-    version: 0.2.3
+    version: 0.3.0
```

Comment:
Maybe it's better to revert this one to 0.2.3 just for now as it needs other changes as well in the values file (also the `Chart.lock` file needs an update for example with `helm dependency update charts/drax`). Details in #252 (reminds me btw to bump the sub-charts over there still, added it as a task in that PR).